t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -308,11 +284,7 @@ sub map_elements ($selector, $mapping) {
     return $_driver->execute_script("return Array.from(document.querySelectorAll('$selector')).map(e => [$mapping]);");
 }
 
-sub wait_until {
-    my ($check_function, $check_description, $timeout, $check_interval) = @_;
-    $timeout //= 100;
-    $check_interval //= .1;
-
+sub wait_until ($check_function, $check_description, $timeout = 100, $check_interval = .1) {
```

Comment:
```suggestion
sub wait_until ($check_function, $check_description, $timeout, $check_interval = .1) {
    $imteout //= 100;
```

because

    Use of uninitialized value $timeout in numeric le (<=) at /home/squamata/project/t/ui/../lib/OpenQA/SeleniumTest.pm line 293